lang/en/main.json

```diff
@@ -5,7 +5,6 @@
y choosing LibreSign, your company is investing in a solution that places security and privacy at the center of everything we do.\n\n## Who is LibreSign for?\n\nLibreSign is a versatile solution developed to meet the specific needs of various sectors, including government, education, and corporate enterprises. By simplifying and streamlining processes such as the signing of government contracts, academic documents, and corporate agreements, LibreSign promotes efficiency at all levels of these institutions.\n\nBy using LibreSign, businesses are not only investing in a practical tool; they are joining a revolution that redefines and optimizes process management. This transformation aims not only to enhance efficiency but also to strengthen security, taking them to the highest level of excellence.": "Agility and security in business transactions are essential for the success of any company. It is in this scenario that LibreSign stands out as the smart choice for businesses across various sectors, offering not only efficiency but also a commitment to security and privacy.\n\nDeveloped by LibreCode, a cooperative of IT professionals, LibreSign is more than just software; it is an expression of the Free and Open-Source Software (FOSS) philosophy, reflecting not only what we do but why we do it. After all, we are driven by the conviction that technology should be accessible to all, promoting the freedom of use, modification, and distribution.\n\nBy adopting the Free and Open-Source model, LibreSign not only provides businesses with a cost-effective solution but also offers the flexibility needed to adapt the software to the specific demands of each business. This means that your company is not just acquiring a product but becoming part of a community dedicated to building advanced technological solutions.\n\n## Security first\n\nDeveloped with the highest standards of security and encryption, our users can sign electronic documents with confidence, knowing that their sensitive information is protected.\n\nEncryption ensures that electronic signatures are secure and inviolable, providing peace of mind not only for businesses but also for customers and partners. By choosing LibreSign, your company is investing in a solution that places security and privacy at the center of everything we do.\n\n## Who is LibreSign for?\n\nLibreSign is a versatile solution developed to meet the specific needs of various sectors, including government, education, and corporate enterprises. By simplifying and streamlining processes such as the signing of government contracts, academic documents, and corporate agreements, LibreSign promotes efficiency at all levels of these institutions.\n\nBy using LibreSign, businesses are not only investing in a practical tool; they are joining a revolution that redefines and optimizes process management. This transformation aims not only to enhance efficiency but also to strengthen security, taking them to the highest level of excellence.",
```

Comment:
This change sounds strange.
Was the file with this content removed in another PR?